keanu-project/src/test/java/io/improbable/keanu/vertices/bool/BoolVertexTest.java

```diff
@@ -1,12 +1,11 @@
 package io.improbable.keanu.vertices.bool;
 
+import static io.improbable.keanu.vertices.bool.BoolVertex.not;
```

Comment:
looks like your IntelliJ settings are different to [mine](https://improbableio.atlassian.net/wiki/spaces/~616259109/pages/283574476/IntelliJ+settings)

Reply:
I've just gone through these and the import stuff is as defined on StackOverflow?